### PR TITLE
del

### PR DESCRIPTION
**Pull request by amiiiiii830**
delelte

### CODE REVIEW

**Review by jaykchen — COMMENTED** [review]
comment on pr from Amy

**Review by jaykchen — CHANGES_REQUESTED** [review]
can I reject it?

**Review by jaykchen — APPROVED** [review]

**Comment by amiiiiii830** [comment]
this is answer to the old thread

**Comment by jaykchen** [comment]
testing new labels

**Review by jaykchen — COMMENTED** [review]
this is a comment to pr review

**Review by jaykchen — COMMENTED** [review]
the latedsct review comment

**Comment by jaykchen** [comment]
> delelte

my comment added

**Review by jaykchen — CHANGES_REQUESTED** [review]
pls delete unwanted file

**Review by jaykchen — COMMENTED** [review]
atomic comment

File: README.md [hunk 1]
```diff
@@ -1,5 +0,0 @@
-# demo
-git lesson
-
-
```
Comment (jaykchen) [inline README.md:4]: PR review comment

Comment (jaykchen) [reply README.md:4]: this is a reply to the comment

File: pages/index.vue [hunk 2]
```diff
@@ -2,6 +2,7 @@
   <div class="page-index">
     <Examples />
   </div>
```
Comment (jaykchen) [inline pages/index.vue:4]: I have some comments

**Review by jaykchen — CHANGES_REQUESTED** [review]
this is new comment

File: pages/index.vue [hunk 3]
```diff
@@ -2,6 +2,7 @@
...
+  <span> some random changes</span>
```
Comment (jaykchen) [inline pages/index.vue:5]: I don't like this addtion

Comment (jaykchen) [reply pages/index.vue:5]: some reply to comment

**Review by jaykchen — APPROVED** [review]

**Comment by jaykchen** [comment]
rejected

**Pull request closed without merging**